OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -102,13 +108,14 @@ void INotifyKilled.Killed(Actor self, AttackInfo e)
 			if (IsTraitDisabled || !self.IsInWorld)
 				return;
 
-			if (self.World.SharedRandom.Next(100) > Info.Chance)
+			var sharedRandom = self.World.SharedRandom.Next(100);
```

Comment:
The name implies that this is the shared random generator, not a random number. Querying a random number is fast, so this likely just serves to make the code harder to understand. Can we revert this?